src/main/java/org/mitre/synthea/world/agents/Payer.java

```diff
@@ -504,10 +558,12 @@ private String getEntryType(Entry entry) {
    * @param key the key (the encounter type and whether it was covered/uncovered)
    */
   private synchronized void incrementEntries(Integer year, String key) {
-    if (!entryUtilization.contains(year, key)) {
-      entryUtilization.put(year, key, new AtomicInteger(0));
+    if (entryUtilization != null) { // TODO remove once entryUtilization is serializable
```

Comment:
Is this a comment that means the check should be removed once `Table` becomes serializable?

Reply:
Yes, good catch.